### ISSUE TITLE
Question: Source for data normalization in Online ASR Microphone demo

### ISSUE THREAD

**Issue by antimora** [NVIDIA/NeMo#567]
The question pertains this demo example. https://github.com/NVIDIA/NeMo/blob/master/examples/asr/notebooks/2_Online_ASR_Microphone_Demo.ipynb

The demo has a bunch of mean and std data points for normalization. How is this data derived? Is this per entire dataset? If it's per dataset, is there any advantage normalizing by dataset instead of per feature or batch?

**Comment by vsl9** [comment]
`fixed_mean` and `fixed_std` were computed as means and standard deviations of mel-spectrogram channels on training dataset. We use 64 mel channels: that's why `fixed_mean` and `fixed_std` have 64 values.
We normalize each example separately during training and offline inference. But online inference works on infinite audio stream, so we need to use either fixed or adaptive normalization. The former is simpler for the notebook.
This notebook is rather a toy example demonstrating online inference. To learn more about optimized and efficient online ASR inference, please sign-up to Jarvis Early Access Program: https://developer.nvidia.com/nvidia-jarvis

**Comment by dangvansam98** [comment]
> `fixed_mean` and `fixed_std` were computed as means and standard deviations of mel-spectrogram channels on training dataset. We use 64 mel channels: that's why `fixed_mean` and `fixed_std` have 64 values.
> We normalize each example separately during training and offline inference. But online inference works on infinite audio stream, so we need to use either fixed or adaptive normalization. The former is simpler for the notebook.
> This notebook is rather a toy example demonstrating online inference. To learn more about optimized and efficient online ASR inference, please sign-up to Jarvis Early Access Program: https://developer.nvidia.com/nvidia-jarvis

how can i to calculate the fixed_mean and fixed_std in my training set? thanks
